esp/platform/espcontext.cpp

```diff
@@ -466,9 +466,6 @@ class CEspContext : public CInterface, implements IEspContext
     {
         if (m_txSummary)
         {
-            if (!m_hasException && (getEspLogLevel() <= LogNormal))
```

Comment:
Discussed with Tony. We may add another configurable variable TxSummaryLevel which gives a developer to control when a TxSummary item should be logged. I will create another JIRA to work on that, as well as a possibility to suppress TxSummary for front-end generated requests.